### DISCUSSION TITLE
How about linking bird species to Wikipedia (or some other resource?)

### DISCUSSION THREAD

**Discussion by kchristensen** [mmcc-xx/BirdCAGE#45] [Ideas]
BirdNET-Pi takes the easy route and just links to `https://wikipedia.com/wiki/<Scientific Name>` [(Example)](https://wikipedia.org/wiki/Myiophobus_cryptoxanthus), which generally redirects to the right place. This is pretty easy to do, but I wasn't going to step on any toes if there was something cooler planned.

**Comment by matthew73210** [comment]
Or ebird ;)

**Comment by mmcc-xx** [comment]
All toes are safe. There's also All About Birds, but the URLs seem to be based on common names. I don't know if it is localized for other languages.

I think eBird has an API for looking up their abbreviated names based on scientific names, e.g. norfli below.

https://www.allaboutbirds.org/guide/Northern_Flicker/
https://ebird.org/species/norfli
https://en.wikipedia.org/wiki/Colaptes_auratus

I'd probably lean toward eBird.

**Comment by grbray** [comment]
I'd love to have a thumbnail of the species too (well, my wife particularly).  That way you hear it and can somewhat easily identify which bird it is without having to do a lookup.

**Comment by matthew73210** [comment]
Having a look at the ebird API and I can already think up some interesting ideas : 
Have the nearest observation of a bird that registers. good for validating if a bird really exists or not
